components/dashboard/src/repositories/detail/prebuilds/PrebuildDetailPage.tsx

```diff
@@ -158,11 +158,11 @@ export const PrebuildDetailPage: FC = () => {
                                             </div>
                                         )}
                                     </div>
-                                    <div className="flex gap-1">
+                                    <div className="flex gap-1 items-center">
                                         <img
                                             className="w-5 h-5 rounded-full"
                                             src={info.prebuild.commit?.author?.avatarUrl}
-                                            alt={info.prebuild.commit?.author?.name}
+                                            alt=""
```

Comment:
didn't understand the intent for this change 🤔 

was it returning the wrong data? If yes, we can still keep it a generic one
```suggestion
                                            alt="User Avatar"
```

Reply:
It was not returning anything incorrect, but in this case, the alt text does not add value to screen readers and can be considered a decorative image. If we keep as-is, we're essentially repeating ourselves

- https://web.dev/shows/http-203/flf2vS0IoRs/
- https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/alt#value